parse_yaml_files.py

```diff
@@ -88,7 +91,10 @@ def read_yml_files():
     problems_url["icon"] = NOT_OK
     df = df.merge(problems_url, how="left", on="name")
 
-    df.to_csv("data/citizen-science-projects.csv")
+    # Save to CSV
+    df.to_csv("data/citizen-science-projects-nl.csv")
+    # Save to Excel
+    df.to_excel("data/citizen-science-projects-nl.xls")
```

Comment:
Should be `.xlsx`!!

Reply:
I reversed it to xls as xlsx was giving me TONS of problems to save/read with Pandas. Not entirely sure why.

Reply:
Complete no-go.